app.py

```diff
@@ -6,20 +6,19 @@
 
```

Comment:
Lines 2 & 5 are obsolete (also the SECRET_KEY can be removed)

Reply:
What did SECRET_KEY used to do? was that for identifying/accessing the session cookies/data?
also I'm not sure line 2 is obsolete, did you mean a different line?

Reply:
SECRET_KEY was used to encrypt the flask session cookie, but we're using it no more: https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY

You're right, we still need line 2 (just renamed). Line 5 can go though (along with `flask_config.py`)